apps/passport-client/new-components/screens/Home/NewHomeScreen.tsx

```diff
@@ -137,7 +131,14 @@ export const PageCircleButton = styled.button<{
 
 const ButtonsContainer = styled.div`
   display: flex;
+  position: absolute;
+  height: ${BUTTONS_CONTAINER_HEIGHT};
```

Comment:
I'm not sure why `BUTTONS_CONTAINER_HEIGHT` is needed but here it does nothing but an invalid on css:
![image](https://github.com/user-attachments/assets/8fd5b029-a9f6-4ecf-818b-5d88ff543269)